### DISCUSSION TITLE
刻度值的参数按照文档写的不管用

### DISCUSSION THREAD

**Discussion by zhiyou1201** [antvis/G2#5631] [Q&A]
- **G2 Version**:
- **Platform**:
- **Mini Showcase(like screenshots)**:
- **CodePen Link**:

**Comment by pearmini** [comment]
提供具体复现代码

**Reply by zhiyou1201** [reply]
![image](https://github.com/antvis/G2/assets/75651528/1cefa42e-64c9-47ac-aaaa-ad15fa4bb315)
![image](https://github.com/antvis/G2/assets/75651528/e35256d5-2857-4779-bd76-0af17de1d0e3)
![image](https://github.com/antvis/G2/assets/75651528/f95c5ab8-7455-4c5c-a082-e44f9d8a621f)
写这些不管用 还是竖着展示

**Reply by pearmini** [reply]
设置 paddingBottom 为一个具体的值

**Comment by zhiyou1201** [comment]
请问能解决吗

**Comment by zhiyou1201** [comment]
![image](https://github.com/antvis/G2/assets/75651528/472c21da-3201-4a8a-90ef-2ee6ce968ac3)
怎么把中间为0的线放到x轴线上   像这样的
![image](https://github.com/antvis/G2/assets/75651528/5aa560e9-d9d3-4c51-90b9-9f30c814ac81)

**Comment by zhiyou1201** [comment]
在麻烦您看看 这个怎么解决  怎么把中间为0的线放到x轴线上

**Reply by pearmini** [reply]
```js
import { Chart } from '@antv/g2';

const chart = new Chart({
  container: 'container',
  autoFit: true,
});

chart
  .line()
  .data([0, 0, 0, 0])
  .encode('x', (_, i) => i)
  .encode('y', (d) => d)
  .scale('y', { domainMin: 0, domainMax: 1 })
  .axis('y', { tickMethod: () => [0] });

chart.render();

```

**Comment by zhiyou1201** [comment]
是0的时候是到下面了  但是有数据的时候 数据都飞了
![image](https://github.com/antvis/G2/assets/75651528/d5de2487-e4aa-4e2c-84b0-033658fbd17a)
正常数据是这样的
![image](https://github.com/antvis/G2/assets/75651528/904795d1-08d8-4d5e-8503-db2c9e28145d)

**Reply by pearmini** [reply]
这个需要判断，如果数据只有 0 的时候就上面那样，否则就正常

**Comment by zhiyou1201** [comment]
![image](https://github.com/antvis/G2/assets/75651528/08ffc891-236f-4fea-a6f0-602c7870ad6d)
这个柱状图没有数据的时候就什么都不展示了
![image](https://github.com/antvis/G2/assets/75651528/3e3059d7-6577-4446-8b1c-485186d64dc5)
能不能没有数据的时候展示这样的 展示出来图的整体 没有数据也可以 把图的整体展示出来
![image](https://github.com/antvis/G2/assets/75651528/116b487b-31a2-4730-acb5-4f9db2ef168c)

**Reply by pearmini** [reply]
可以当数据都为空的 mock 数据实现图中效果，否者正常渲染

**Comment by zhiyou1201** [comment]
请问有解决办法吗
